### DISCUSSION TITLE
can Integration with risingwave

### DISCUSSION THREAD

**Discussion by wacdev** [GreptimeTeam/greptimedb#3573] [Ideas]
### What problem does the new feature solve?

https://github.com/risingwavelabs/risingwave

I want to do some data flow statistics, such as counting the number of impressions/clicks of each picture. It would be great if I could connect to risingwave

我想做一些数据流的统计，比如统计每张图片的展示次数/点击次数，如果能可以连上risingwave就好了

### What does the feature do?

risingwave可以对接postgresql或者mysql ，完善postgresql 或者mysql接口即可
Risingwave can connect to postgresql or mysql, just improve the postgresql or mysql interface

**Comment by sunng87** [comment]
sounds great. Have you tested to connect greptimedb as mysql or postgresql to risingwave? Is there any error that blocks the integration?

**Comment by xxaier** [comment]
this need postgres cdc or mysql cdc
https://www.risingwave.dev/docs/current/ingest-from-postgres-cdc/

PostgreSQL CDC: A Comprehensive Guide
https://www.arcion.io/learn/postgresql-cdc

**Comment by 2547409617** [comment]
Great idea!  I am interested in this feature, however I am not familiar with code structure.
Any suggestions such as control/data flow?
If someone owner it, I am very glad to assist with detailed code work.

**Comment by tisonkun** [comment]
Move to GH Discussion forum "Ideas" until we have some concrete starting points.
